Dockerfile

```diff
@@ -30,4 +30,4 @@ RUN composer install --no-interaction
 
 WORKDIR /home
 RUN du -sh *
-CMD python -u ./src/main.py --data=/data
+CMD ./src/run.sh
```

Comment:
Segmentation fault je fatal error, neda sa v Pythone odchytit, preto je to obalene do bash scriptu.